EKF/vel_pos_fusion.cpp

```diff
@@ -45,119 +45,136 @@
 #include <ecl.h>
 #include <mathlib/mathlib.h>
 
-void Ekf::fuseVelPosHeight()
+/**
+ * Update the EKF state with velocity and position measurements.
+ *
+ * @param innov  	Input/Output	Innovation which is fuse_mask is set to false or do not pass
```

Comment:
Can you please reword the description for this. I have read it a few times and am not sure what it means.

Reply:
You are totally right, this sentence does not make any sense. Thanks for noticing.

Reply:
I think this line was part of an outdated version. Please have a look at the newest version.